serf/config.go

```diff
@@ -253,6 +257,11 @@ type Config struct {
 	//
 	// WARNING: this should ONLY be used in tests
 	messageDropper func(typ messageType) bool
+
+	//ValidateNodeNames specifies whether or not nodenames should
+	// be alphanumeric and within 128 characters
+	//TODO(schristoff): should this be here?
```

Comment:
I think this is the right place for this setting - this config controls how the serf instance behaves